client/tests/app.spec.js

```diff
@@ -1,7 +0,0 @@
-'use strict'; //eslint-disable-line strict
```

Comment:
This file was moved to the `tests` directory to live alongside the other test files